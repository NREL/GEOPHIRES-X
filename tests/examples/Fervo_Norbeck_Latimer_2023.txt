--- Case Study: Fervo Energy Whitepaper
--- Norbeck J.H. and Latimer T.M. 2023.
---   "Commercial-Scale Demonstration of a First-of-a-Kind Enhanced Geothermal System."
---   https://doi.org/10.31223/X52X0B
---  Drilling Price paper:
---     "Review Of Drilling Performance In A Horizontal EGS Development"
---     Kareem El-Sadi Brittany Gierke Elliot Howard Christian Gradl
---     https://pangea.stanford.edu/ERE/db/GeoConf/papers/SGW/2024/Elsadi.pdf
--- Looking to make:
---         gross power production of 2 to 3.5 MWe
---         production temperature of 336 degF (169 degC)
---         pumping power 500 to 1000 kW

***Reservoir Parameters***
Reservoir Model, 1
Reservoir Volume Option,1,					---Should be 1 2 3 or 4. See manual for details.
Reservoir Density, 2800, per the paper phyllite + quartzite + diorite + granodiorite,
Reservoir Depth, 7700 feet, per the paper
Reservoir Heat Capacity,1000,					---[J/kg/K]
Reservoir Thermal Conductivity, 2.7
Number of Fractures, 100, per the paper
Fracture Shape, 4, per the paper
Fracture Height, 160, per the paper
Fracture Width, 160, per the paper (distance between wellbores assumes a dipole flow field)
Fracture Separation, 10 meter
Number of Segments,1
Gradient 1, 76.74, per the paper
Plant Outlet Pressure, 1500 psi, per the paper

*** Wellbore Parameters***
Number of Injection Wells, 1,					---[-]
Number of Production Wells,1,					---[-]
Well Geometry Configuration, 4,       ----L
Has Nonvertical Section, True
Multilaterals Cased, True
Number of Multilateral Sections, 2,  ---Two parallel horizontal sections
Total Nonvertical Length, 3250 feet, per the paper
Well Drilling Cost Correlation,10,	per the drill cost paper - works out to $400/ft
Horizontal Well Drilling Cost Correlation,10,	per the drill cost paper - works out to $400/ft
Production Flow Rate per Well, 41.02, =650 gpm per the paper - per the paper the maximum flow rate was 63 L/s but the range was 550-750 gpm
Production Well Diameter, 7, per the paper
Injection Well Diameter, 7, per the paper
Well Separation, 365 feet, per the paper
Injection Temperature, 45 degC, per the paper 75 to 125 degF
Injection Wellbore Temperature Gain,3,				---[deg.C]
Reservoir Impedance, 0.33, per paper, matching pumping power report 500-1000 kW
Ramey Production Wellbore Model,1,				---Should be 0 (disabled) or 1 (enabled)
Utilization Factor,.9,						---[-]Water Loss Fraction, .02x
Water Loss Fraction,0.02,						---[-]
Maximum Drawdown,1,						---[-] no redrilling considered

***Surfaceplant Parameters***
Ambient Temperature, 8 degC, per the paper. Published ambient is 10.27 degC
Surface Temperature, 8 degC, per the paper. Published ambient is 10.27 degC
End-Use Option,1,						---electricity as the main product
Plant Lifetime, 10
Power Plant Type,2,						---supercritical binary cycle
End-Use Efficiency Factor,.9,					---[-]
Circulation Pump Efficiency, 0.80, per the paper

*** Economic Parameters***
Economic Model,3,						---BICYCLE Levelized Cost Model
Starting Electricity Sale Price,0.15
Ending Electricity Sale Price,1.00, essentially no limit s the rate can rise continuously with inflation
Electricity Escalation Rate Per Year, 0.004053223, End up being about 2% inflation
Electricity Escalation Start Year,1
Fraction of Investment in Bonds,.5,				---[-] Required for BICYCLE model
Combined Income Tax Rate,.3,					---[-] Required for BICYCLE model
Gross Revenue Tax Rate,0,					---[-] Required for BICYCLE model
Inflated Bond Interest Rate,.05,				---[-] Required for BICYCLE model
Inflated Equity Interest Rate,.08,				---[-] Required for BICYCLE model
Inflation Rate,.02,						---[-] Required for BICYCLE model
Investment Tax Credit Rate,0,					---[-] Required for BICYCLE model
Inflation Rate During Construction, 0.05,				---[-]
Property Tax Rate,0,						---[-] Required for BICYCLE model

***Other Parameters***
Print Output to Console, 1,					---Should be 1 (to print) or 0 (to not print)
Time steps per year,10,						---[-]
