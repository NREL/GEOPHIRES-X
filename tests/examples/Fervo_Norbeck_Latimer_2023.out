                               *****************
                               ***CASE REPORT***
                               *****************

Simulation Metadata
----------------------
 GEOPHIRES Version: 3.4.29
 Simulation Date: 2024-05-15
 Simulation Time:  11:32
 Calculation Time:      0.349 sec

                           ***SUMMARY OF RESULTS***

      End-Use Option: Electricity
      Average Net Electricity Production:                     2.63 MW
      Electricity breakeven price:                           24.23 cents/kWh
      Number of production wells:                             1
      Number of injection wells:                              1
      Flowrate per production well:                          41.0 kg/sec
      Well depth (or total length, if not vertical):          2.3 kilometer
      Geothermal gradient:                                    0.0767 degC/m


                           ***ECONOMIC PARAMETERS***

      Economic Model  = BICYCLE
      Accrued financing during construction:                  5.00
      Project lifetime:                                      10 yr
      Capacity factor:                                       90.0 %
      Project NPV:                                         -10.73 MUSD
      Project IRR:                                          -2.24 %
      Project VIR=PI=PIR:                                    0.64
      Project MOIC:                                         -0.10
      Project Payback Period:                          N/A

                          ***ENGINEERING PARAMETERS***

      Number of Production Wells:                             1
      Number of Injection Wells:                              1
      Well depth (or total length, if not vertical):          2.3 kilometer
      Water loss rate:                                        2.0
      Pump efficiency:                                       80.0
      Injection temperature:                                 48.0 degC
      Production Wellbore heat transmission calculated with Ramey's model
      Average production well temperature drop:               4.1 degC
      Flowrate per production well:                          41.0 kg/sec
      Injection well casing ID:                               7.000 in
      Production well casing ID:                              7.000 in
      Number of times redrilling:                             0
      Power plant type:                                       Supercritical ORC


                         ***RESOURCE CHARACTERISTICS***

      Maximum reservoir temperature:                        400.0 degC
      Number of segments:                                     1
      Geothermal gradient:                                    0.0767 degC/m


                           ***RESERVOIR PARAMETERS***

      Reservoir Model = Multiple Parallel Fractures Model
      Bottom-hole temperature:                              188.11 degC
      Fracture model = Rectangular
      Well separation: fracture height:                     160.00 meter
      Fracture width:                                                 160.00 meter
      Fracture area:                                      25600.00 m**2
      Reservoir volume calculated with fracture separation and number of fractures as input
      Number of fractures:                                  100.00
      Fracture separation:                                   10.00 meter
      Reservoir volume:                                25344000 m**3
      Reservoir impedance:                                    0.33 GPa.s/m**3
      Reservoir density:                                   2800.00 kg/m**3
      Reservoir thermal conductivity:                         2.70 W/m/K
      Reservoir heat capacity:                             1000.00 J/kg/K


                           ***RESERVOIR SIMULATION RESULTS***

      Maximum Production Temperature:                       184.1 degC
      Average Production Temperature:                       182.5 degC
      Minimum Production Temperature:                       170.2 degC
      Initial Production Temperature:                       181.8 degC
      Average Reservoir Heat Extraction:                     23.03 MW
      Production Wellbore Heat Transmission Model = Ramey Model
      Average Production Well Temperature Drop:               4.1 degC
      Total Average Pressure Drop:                        12378.0 kPa
      Average Injection Well Pressure Drop:                 337.8 kPa
      Average Reservoir Pressure Drop:                    14070.0 kPa
      Average Production Well Pressure Drop:                350.0 kPa
      Average Buoyancy Pressure Drop:                     -2379.9 kPa


                          ***CAPITAL COSTS (M$)***

         Drilling and completion costs:                      10.21 MUSD
             Drilling and completion costs per vertical production well:         3.06 MUSD
             Drilling and completion costs per vertical injection well:          3.06 MUSD
             Drilling and completion costs per non-vertical sections:            3.61 MUSD
         Stimulation costs:                                   1.51 MUSD
         Surface power plant costs:                          12.64 MUSD
         Field gathering system costs:                        1.52 MUSD
         Total surface equipment costs:                      14.16 MUSD
         Exploration costs:                                   3.65 MUSD
      Total capital costs:                                   29.53 MUSD


                ***OPERATING AND MAINTENANCE COSTS (M$/yr)***

         Wellfield maintenance costs:                         0.23 MUSD/yr
         Power plant maintenance costs:                       0.53 MUSD/yr
         Water costs:                                         0.02 MUSD/yr
      Total operating and maintenance costs:                  0.78 MUSD/yr


                           ***SURFACE EQUIPMENT SIMULATION RESULTS***

      Initial geofluid availability:                          0.16 MW/(kg/s)
      Maximum Total Electricity Generation:                   3.36 MW
      Average Total Electricity Generation:                   3.28 MW
      Minimum Total Electricity Generation:                   2.73 MW
      Initial Total Electricity Generation:                   3.25 MW
      Maximum Net Electricity Generation:                     2.71 MW
      Average Net Electricity Generation:                     2.63 MW
      Minimum Net Electricity Generation:                     2.06 MW
      Initial Net Electricity Generation:                     2.60 MW
      Average Annual Total Electricity Generation:           25.64 GWh
      Average Annual Net Electricity Generation:             20.58 GWh
      Initial pumping power/net installed power:             24.86 %
      Average Pumping Power:                                  0.65 MW

                            ************************************************************
                            *  HEATING, COOLING AND/OR ELECTRICITY PRODUCTION PROFILE  *
                            ************************************************************
  YEAR       THERMAL               GEOFLUID               PUMP               NET               FIRST LAW
             DRAWDOWN             TEMPERATURE             POWER             POWER              EFFICIENCY
                                     (degC)               (MW)              (MW)                  (%)
   1           1.0000                181.79               0.6470            2.6029               11.3571
   2           1.0090                183.42               0.6465            2.6796               11.5513
   3           1.0108                183.76               0.6464            2.6956               11.5913
   4           1.0118                183.93               0.6464            2.7040               11.6121
   5           1.0124                184.05               0.6463            2.7095               11.6258
   6           1.0129                184.13               0.6463            2.7133               11.6353
   7           1.0129                184.13               0.6464            2.7134               11.6354
   8           1.0109                183.77               0.6469            2.6960               11.5914
   9           1.0023                182.21               0.6489            2.6205               11.3981
  10           0.9785                177.88               0.6543            2.4152               10.8550


                              *******************************************************************
                              *  ANNUAL HEATING, COOLING AND/OR ELECTRICITY PRODUCTION PROFILE  *
                              *******************************************************************
  YEAR             ELECTRICITY                   HEAT                RESERVOIR            PERCENTAGE OF
                    PROVIDED                   EXTRACTED            HEAT CONTENT        TOTAL HEAT MINED
                   (GWh/year)                  (GWh/year)            (10^15 J)                 (%)
   1                  20.9                       182.1                  9.29                   6.59
   2                  21.2                       183.2                  8.63                  13.23
   3                  21.3                       183.5                  7.97                  19.87
   4                  21.3                       183.7                  7.31                  26.52
   5                  21.4                       183.8                  6.64                  33.17
   6                  21.4                       183.9                  5.98                  39.83
   7                  21.3                       183.7                  5.32                  46.48
   8                  21.0                       182.5                  4.66                  53.09
   9                  20.0                       178.7                  4.02                  59.56
  10                  16.0                       153.6                  3.47                  65.13


                             ********************************
                             *  REVENUE & CASHFLOW PROFILE  *
                             ********************************
Year            Electricity             |            Heat                  |           Cooling                 |         Carbon                    |          Project
Since     Price   Ann. Rev.  Cumm. Rev. |   Price   Ann. Rev.   Cumm. Rev. |  Price   Ann. Rev.   Cumm. Rev.   |   Price   Ann. Rev.   Cumm. Rev.  | OPEX    Net Rev.      Net Cashflow
Start    (cents/kWh)(MUSD/yr) (MUSD)    |(cents/kWh) (MUSD/yr)    (MUSD)   |(cents/kWh) (MUSD/yr)    (MUSD)    |(USD/tonne) (MUSD/yr)    (MUSD)    |(MUSD/yr) (MUSD/yr)    (MUSD)
________________________________________________________________________________________________________________________________________________________________________________________
  1      0.00          -29.53   0.00     |    0.00     0.00         0.00    |    0.00     0.00         0.00     |    0.00     0.00         0.00     |  0.00     -29.53     -29.53
  2     15.00           2.36   3.14     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  0.78      2.36     -27.17
  3     15.00           2.40   6.32     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  0.78      2.40     -24.77
  4     15.41           2.50   9.60     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  0.78      2.50     -22.27
  5     15.81           2.59  12.97     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  0.78      2.59     -19.68
  6     16.22           2.69  16.44     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  0.78      2.69     -16.99
  7     16.62           2.78  19.99     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  0.78      2.78     -14.21
  8     17.03           2.85  23.63     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  0.78      2.85     -11.36
  9     17.43           2.88  27.29     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  0.78      2.88     -8.47
 10     17.84           2.78  30.85     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  0.78      2.78     -5.69
